convert2rhel/special_cases.py

```diff
@@ -101,25 +101,34 @@ def perform_java_openjdk_workaround():
 
 
 def unprotect_shim_x64():
-    """Remove the shim-x64 package yum protection on UEFI-based Oracle Linux 7 systems as it causes a yum traceback.
+    """Remove the shim-x64 package yum protection on UEFI-based Oracle Linux 7
+    systems as it causes a yum traceback.
```

Comment:
Minor note that doesn't need to be done at all; just bringing it up since this is a change:

Standard docstring format differs from this in two ways:

* The summary should be on a single line rather than two.
* When a docstring takes up more than one line, it should start on the line after the `"""`.
We don't extract docs from the docstrings so it's okay that these aren't formatted that way; just mentioning it for your information

Reply:
Glad that you picked that up! I can fix it since it's an easy change to the docstring. I will do that tomorrow.

Reply:
Done!